test/fixtures/test-runner/bail/bail.js

```diff
@@ -0,0 +1,16 @@
+const test = require('node:test');
+
+test('nested', (t) => {
+  t.test('first', () => {});
+  t.test('second', () => {
+    throw new Error();
+  });
+  t.test('third', () => {});
```

Comment:
```suggestion
  t.test('third', common.mustNotCall(() => {});
```

Reply:
We check that is called when `bail` is not activated

Reply:
so that we make sure on `process.beforeExit` that it really was not called. and not when reporting the test